### ISSUE TITLE
xterm flickers with gnupg.vim 2.6 

### ISSUE THREAD

**Issue by ao2** [jamessan/vim-gnupg#64]
Hi,

I updated from `gnupg.vim` 2.5 to 2.6 and I started to notice that xterm flickers when opening a .gpg file with vim, before showing the key prompt from gpg. The issue is a lot more visible with a full screen xterm.

I do not see the issue with other terminal emulators, so it might be a xterm+vim issue which `gnupg.vim` triggers; I bisected the history and I found out that the issue has been introduced in commit bd3ebdf from issue #17.

For now I can live with just commenting the term rediscovery workaround, but I wanted tor report that the workaround from commit  bd3ebdf may have side effects like the one I am experiencing.

Thanks,
   Antonio

**Comment by jamessan** [comment]
Yeah, I know there are side effects, but a flicker in the redraw is better than not being able to use various keys on your keyboard.

**Comment by ao2** [comment]
@jamessan if there is no easy solution could we make the workaround optional, guarding it with a config variable?
Even with a default to ON it'd be OK for me, so I can at least disable it via `.vimrc` instead of patching the plugin.

I can send  a patch for that if the approach is fine with you.

Thanks,
   Antonio

**Comment by ao2** [comment]
Just for the records, by reverting the workaround from https://github.com/jamessan/vim-gnupg/commit/bd3ebdff6cb18d09c4bdbb72e28e07841c5076eb and using `pinentry-tty` I avoid both the issue from https://github.com/jamessan/vim-gnupg/issues/17 and the flickering from this issue.
